code/_helpers/global_lists.dm

```diff
@@ -44,6 +44,11 @@ var/global/list/poster_designs = list()
 // Uplinks
 var/list/obj/item/device/uplink/world_uplinks = list()
 
+
+//Neotheology
+var/global/list/all_rituals = list() //List of all rituals
+var/list/global_ritual_cooldowns = list() // internal lists. Use ritual's cooldown_category
```

Comment:
"global ritual cooldown" list which is not global :ohmy:
Is it supposed to be non global?

Reply:
its declared in global scope, works fine,ehh